web/modules/custom/collection/collection.module

```diff
@@ -81,24 +104,90 @@ function collection_entity_insert(EntityInterface $entity) {
 }
 
 /**
+ * Implements hook_ENTITY_TYPE_access().
+ *
+ * Interrupts the permissions for updating a collection. The permission is not
+ * granted if there are no transitions allowed for the current state of the
+ * node.
+ */
+function collection_rdf_entity_access(EntityInterface $entity, $operation, AccountInterface $account) {
+  if ($entity->bundle() != 'collection' || $operation != 'view') {
```

Comment:
Hm. Now on $operation == 'edit' will exit here. I we should have here next "OR" chained conditions:
- `$entity->bundle() != 'collection'`
- `$account->isAnonymous()` (taken from `collection_entity_access()`)
- `\Drupal::currentUser()->hasPermission('administer rdf entity')`. Shouldn't this be in effect also for `view` op? If yes, you should move it here and remove it from `edit`.